### PR TITLE
Ignore warnings

### PR DESCRIPTION
**Pull request by OzymandiasTheGreat**
As we discussed in #6, added `ignoreWarnings` parameter to `bundle()` and set `const ignoredWarnings = ["UNRESOLVED_IMPORT"]` so now it possible to set warnings ignored by default as well as on the fly.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ const endent = require("endent");
 
 const createPlugin = require("..");
 
-async function bundle(input, output, options = {}) {
+async function bundle(input, output, options = {}, ignoreWarnings = []) {
```
Comment (eight04) [inline test/test.js]: We have too many parameters. It should be better to use:
```js
bundle(input, output, {ignoreWarnings = [], ...options} = {})
```

File: test/test.js [hunk 2]
```diff
@@ -7,7 +7,9 @@ const endent = require("endent");
...
+  const ignoredWarnings = ["UNRESOLVED_IMPORT"];
```
Comment (eight04) [inline test/test.js]: I think `"UNRESOLVED_IMPORT"` should also be passed from each test i.e. all types of warnings will trigger an error by default.

File: test/test.js [hunk 3]
```diff
@@ -19,9 +21,8 @@ async function bundle(input, output, options = {}) {
       createPlugin(options)
     ],
     onwarn(warn) {
-      if (warn.code === "UNRESOLVED_IMPORT") {
-        return;
-      } else if (warn.code === "EMPTY_BUNDLE") {
+      ignoreWarnings.push(...ignoredWarnings);
```
Comment (eight04) [inline test/test.js]: This line will push a new item into ` ignoreWarnings` each time the `warn` function is called.

Comment (OzymandiasTheGreat) [reply test/test.js]: OK, I haven't had enough caffeine yet. Sorry 'bout this.

**Pull request merged**